### ISSUE TITLE
Keras Dropout and BatchNormalization Layer

### ISSUE THREAD

**Issue by ErmiaAzarkhalili** [blei-lab/edward#730]
Hi Dustin,
Could you please provide a real example of NN crated in keras and consists of Dropout and BatchNormalization layers and show how we should feed into Edward to distinguish such layers in Training and Testing times.

I mean how we make Edward to understand Dropout and other layers utilized only in training time? could you explain this concept by example?

**Comment by dustinvtran** [comment]
This is more of a Keras / TF question than one for Edward. I recommend reading [Keras as a simplified interface to TensorFlow: tutorial](https://blog.keras.io/keras-as-a-simplified-interface-to-tensorflow-tutorial.html) if you haven't already. To make Edward utilize Keras layers which differ in training vs test, pass in `K.learning_phase(): 0` (testing) or `K.learning_phase(): 1` (training) in the feed dict during session runs.

**Comment by ErmiaAzarkhalili** [comment]
Hi Dustin.
Thanks for your comment, but, I have already solved the issue (the solution is as exactly as you said). Also, thanks for the link, it was great. I had read it long ago, but it was an excellent opportunity to review it again.

As I told before, utilizing all such layers (Embedding, Dropout, BatchNromalization, etc), I am implementing all the example by only pure Keras. I will certainly share it by all Edward users as soon as it would be ready.
